managed-services/transactional-email/reference-content/dmarc-configuration.mdx

```diff
@@ -0,0 +1,135 @@
+
+### Configuration for strong email security
+
+The following DMARC configuration allows you to prioritize email security, and take strong action against fraudulent or unauthorized emails.
+
+```
+v=DMARC1; p=reject; rua=mailto:mydomain@example.com; ruf=mailto:mydomain-forensic@example.com
+```
+
+- `v=DMARC1` indicates the version of the DMARC protocol being used (version 1).
+-  `p=reject` sets the DMARC policy to "reject", meaning that suspicious emails should be rejected outright and not delivered to the recipient's inbox.
+- `rua=mailto:mydomain@example.com` specifies the email address to which aggregate reports should be sent.
+- `ruf=mailto:mydomain-forensic@example.com` specifies the email address to which forensic reports should be sent.
+
+This configuration might be useful if you want to ensure that only legitimate emails from authenticated sources are delivered to recipients' inboxes, while providing valuable reporting data to monitor and improve email authentication practices.
+
+### Strict configuration
+
+The following DMARC configuration shows you how to enforce strict email authentication policies with robust reporting mechanisms.
+
+    ```
+    v=DMARC1; p=reject; rua=mailto:mydomain@example.com; sp=reject; pct=80; fo=1; adkim=s; aspf=s
+    ```
+
+- `v=DMARC1` indicates the version of the DMARC protocol being used (version 1).
+-  `p=reject` sets the DMARC policy to "reject", meaning that suspicious emails should be rejected outright and not delivered to the recipient's inbox.
+- `rua=mailto:mydomain@example.com` specifies the email address to which aggregate reports should be sent.
+- `sp=reject` sets the subdomain policy to "reject," meaning that emails failing DMARC checks for subdomains should be rejected.
+- `pct=80` specifies that the DMARC policy should be applied to 80% of incoming emails.
+- `fo=1` sets the failure options to "1" meaning that suspicious emails should be treated with caution.
+- `adkim=s` sets the DKIM alignment mode to strict. This means that you are requiring that the DKIM authentication mechanism passes with exact alignment, to pass DMARC authentication. This provides a higher level of assurance that the email originated from the claimed sender and has not been tampered with in transit.
+- `aspf=s` sets the SPF alignment mode to strict. This means that you are requiring that the SPF authentication mechanism passes with exact alignment, to pass DMARC authentication. This provides a higher level of assurance that the email originated from the claimed sender and has not been tampered with in transit.
+
+This configuration might be useful if you want to enforce strict authentication policies while also maintaining visibility into email traffic through comprehensive reporting.
+It ensures that only legitimate emails from authenticated sources are delivered while providing insights into potential authentication failures and security threats.
```

Comment:
I think you can add (or edit an existing scenario) with multiple rua or ruf. Like this:

`rua=mailto:mydomain@example.com,mailto:mydomain-dev@example.com`